### PR TITLE
chore(rds): add oracle versions

### LINKED ISSUE #29360: (aws_rds): (Add Support for RDS Oracle 19.0.0.0.ru-2024-01.rur-2024-01.r1 and 21.0.0.0.ru-2024-01.rur-2024-01.r1 in OracleEngineVersion)
### Describe the feature

As per the RDS Doc: both RDS Oracle 19.0.0.0.ru-2024-01.rur-2024-01.r1 and 21.0.0.0.ru-2024-01.rur-2024-01.r1 are supported.

https://docs.aws.amazon.com/AmazonRDS/latest/OracleReleaseNotes/oracle-version-21-0.html#oracle-version-RU-RUR.21.0.0.0.ru-2024-01.rur-2024-01.r1
https://docs.aws.amazon.com/AmazonRDS/latest/OracleReleaseNotes/oracle-version-19-0.html#oracle-version-RU-RUR.19.0.0.0.ru-2024-01.rur-2024-01.r1

However the same is not reflected in the [OracleEngineVersion](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.OracleEngineVersion.html#class-oracleengineversion) class.

Hence requesting to add the support for the same.

### Use Case

To be able to define RDS Oracle 19.0.0.0.ru-2024-01.rur-2024-01.r1 and 21.0.0.0.ru-2024-01.rur-2024-01.r1 via Static property value like the rest of the RDS Oracle versions.

Workaround: [static of(oracleFullVersion, oracleMajorVersion)](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.OracleEngineVersion.html#static-oforaclefullversion-oraclemajorversion) is know to us.

### Proposed Solution

_No response_

### Other Information

_No response_

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.131.0

### Environment details (OS name and version, etc.)

Mac

### PR DESCRIPTION
**Pull request by ayush-shah-1501**
### Issue # (if applicable)

Closes #29360.

### Reason for this change

Add new two missing oracle version

### Checklist
- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk-lib/aws-rds/lib/instance-engine.ts [hunk 1]
```diff
@@ -1621,6 +1621,10 @@ export class OracleEngineVersion {
   public static readonly VER_21_0_0_0_2023_07_R1 = OracleEngineVersion.of('21.0.0.0.ru-2023-07.rur-2023-07.r1', '21');
   /** Version "21.0.0.0.ru-2023-10.rur-2023-10.r1". */
   public static readonly VER_21_0_0_0_2023_10_R1 = OracleEngineVersion.of('21.0.0.0.ru-2023-10.rur-2023-10.r1', '21');
+  /** Version "19.0.0.0.ru-2024-01.rur-2024-01.r1". */
+  public static readonly VER_19_0_0_0_2024_01_R1 = OracleEngineVersion.of('19.0.0.0.ru-2024-01.rur-2024-01.r1', '19');
```
Comment (watany-dev) [inline packages/aws-cdk-lib/aws-rds/lib/instance-engine.ts:1625]: This should be added to Version "19”

**Comment by watany-dev** [comment]
The comment has been resolved and I don't see any problem with it.

**Comment by nmussy** [comment]
The v19 and v21 `2023-01.r2` releases are also missing, if you want to add them as well:

```ts
  /** Version "19.0.0.0.ru-2023-01.rur-2023-01.r2". */
  public static readonly VER_19_0_0_0_2023_01_R2 = OracleEngineVersion.of('19.0.0.0.ru-2023-01.rur-2023-01.r2', '19');
// ...
  /** Version "21.0.0.0.ru-2023-01.rur-2023-01.r2". */
  public static readonly VER_21_0_0_0_2023_01_R2 = OracleEngineVersion.of('21.0.0.0.ru-2023-01.rur-2023-01.r2', '21');
```

**Comment by paulhcsun** [comment]
Closing this in favor of in favor of https://github.com/aws/aws-cdk/pull/29422

Appreciate the contribution!

**Pull request closed without merging**